### ISSUE TITLE
Can't recursively concat

### ISSUE THREAD

**Issue by rjbergerud** [harttle/liquidjs#44]
Is this the expected behaviour for liquid, to not be able to assign recursively?
```
{% assign yes = "yes" | split: "," %}
    <h5>Does concat work?</h5>
    {% assign yes = yes | concat: yes %}
    {{ yes }}
```
Prints out ["yes"], as opposed to ["yes", "yes"]

**Comment by rjbergerud** [comment]
It seems like I can assign recursively with other filters, so it seems like this is specifically an issue with `concat` or arrays.

**Comment by harttle** [comment]
Seems to be a bug. Maybe the operator precedence isn't handled correctly.

**Comment by rjbergerud** [comment]
I'm trying to find where concat is defined, I don't see it in filter.js.  Where is it?

**Comment by rjbergerud** [comment]
Does it just use the Array.prototype.concat fnct, hence why I can't find it?

**Comment by harttle** [comment]
It's because not implemented. Now it's available in 2.0.0 by [this commit](https://github.com/harttle/liquidjs/commit/485424cc8917459a8d6eb1b36de7124c7acdd952).

> Note: 2.0.0 may not be backward-compatible with 1.x, mainly in whitespace control options.

**Comment by rjbergerud** [comment]
Thanks!
